### ISSUE TITLE
Method [validateUniques] does not exist.

### ISSUE THREAD

**Issue by edno** [edno/kleis#14]
In gitlab by @ValentinGuerlesquin on Aug 11, 2016, 00:40

A la création d'un compte, erreur : 

```
Whoops, looks like something went wrong.

1/1
BadMethodCallException in Validator.php line 3275:
Method [validateUniques] does not exist.
in Validator.php line 3275
at Validator->__call('validateUniques', array('netlogin', 'valgue8547', array('accounts'), object(Validator))) in Validator.php line 485
at Validator->validateUniques('netlogin', 'valgue8547', array('accounts'), object(Validator)) in Validator.php line 485
at Validator->validate('netlogin', 'uniques:accounts') in Validator.php line 425
at Validator->passes() in Validator.php line 450
at Validator->fails() in ValidatesRequests.php line 53
at Controller->validate(object(Request), array('firstname' => 'required|alpha_num|min:3|max:100', 'lastname' => 'required|alpha_num|min:3|max:100', 'netlogin' => 'required|uniques:accounts', 'netpass' => 'required', 'expirydate' => 'required_if:status,1|date', 'category' => 'required', 'status' => 'required|in:0,1', 'group' => 'required|exists:groups,id')) in AccountController.php line 83
at AccountController->addAccount(object(Request))
at call_user_func_array(array(object(AccountController), 'addAccount'), array(object(Request))) in Controller.php line 80
at Controller->callAction('addAccount', array(object(Request))) in
```

Avec RC2

**Comment by edno** [comment]
In gitlab by @edno on Aug 11, 2016, 13:36

Status changed to closed by commit 387dab189a340573d245e3e860c6e15a4ead09f1

**Comment by edno** [comment]
In gitlab by @edno on Aug 11, 2016, 13:41

Corrigé avec le commit 387dab189a340573d245e3e860c6e15a4ead09f1 disponible dans la [RC3](https://gitlab.com/edno/kleis/tags/releasecandidate-3)
